tests/functional/object/mcg/test_s3_vector.py

```
@@ -233,3 +234,400 @@ def test_s3_vector_operations(self, vector_bucket_factory, mcg_obj):
+            assert (
+                index_info.get("ResponseMetadata", {}).get("HTTPStatusCode") == 200
+            ), f"Get operation failed for {index_name}: {index_info}"
+            index_details = index_info.get("index", {})
+            logger.info(
+                f"✓ Retrieved info for {index_name}: dimension={index_details.get('dimension')}, "
+                f"dataType={index_details.get('dataType')}, distanceMetric={index_details.get('distanceMetric')}"
+            )
+
+        # Step 6: Delete a specific index
+        logger.info("Deleting a specific index from the list")
+        index_to_delete = random.choice(valid_index_names)
```

Ruff (0.15.15)

[error] Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)